docs/.vuepress/config.js

```diff
@@ -165,12 +165,12 @@ module.exports = {
         collapsable: true,
         sidebarDepth: 1,
         children: [
-          ['/testnet-plasma', 'PlasmaChain Testnets'],
-          ['/mainnet-plasma', 'PlasmaChain MainNet'],
+          ['/testnet-plasma', 'Loom Testnet'],
+          ['/mainnet-plasma', 'Basechain'],
           ['/validator', 'Become Validator'],
           ['/non-validator-node', 'Non Validator Node'],
           ['/jump-start-plasma', 'Basechain Jump Start'],
-          ['/jump-start-extdev-plasma-us1', 'Extdev Jump Start'],
+          ['/jump-start-extdev-plasma-us1', 'Loom Testnet Jump Start'],
```

Comment:
The title no longer matches the page title

Reply:
> The title no longer matches the page title

Right. Now it's updated.